frontend/src/config/jordan/prism.json

```diff
@@ -16,7 +16,7 @@
     "wms": ["https://api.earthobservation.vam.wfp.org/ows/wms"]
   },
   "map": {
-    "boundingBox": [34.42, 28.36, 39.82, 34.16]
+    "boundingBox": [35.42, 29.36, 38.82, 33.16]
```

Comment:
Is this needed? Larger bounding boxes are preferred when possible since we sometimes use them to get data and bounding boxes too close to the true bbox can create holes.

Reply:
Tagging in @wadhwamatic to make sure he's see this question

Reply:
@ericboucher - I went ahead and reverted this. Jordan appears more zoomed out than needed. I was hoping the zoom attribute would let me fix this but bbox takes precedence. It seems we should allow for more control over the zoom level that won't cause data issues.